src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -126,8 +130,14 @@ export function formatOverlappingForParentAttribute(category: any) {
 }
 
 export function formatOverlapping() {
-    const chartHeight = get(this, "chart.chartHeight", 1);
     const categoriesCount = get(this, "axis.categories", []).length;
+    if (categoriesCount === 1) {
+        // Let the width be auto to make sure "this.value" is displayed on screen
+        return (
+            `<div align="center" style="overflow: hidden; text-overflow: ellipsis">` + this.value + "</div>"
```

Comment:
We should use the template literals here:
```javascript
return (`<div align="center" style="overflow: hidden; text-overflow: ellipsis">${this.value}</div>`);
```

Reply:
Fixed!